zulipterminal/ui_tools/views.py

```diff
@@ -339,3 +341,38 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
             self.view.controller.loop.draw_screen()
             return key
         return super(RightColumnView, self).keypress(size, key)
+
+
+class HelpView(urwid.ListBox):
+    def __init__(self, controller: Any) -> None:
+        self.controller = controller
+        self.log = urwid.SimpleFocusListWalker([
+            urwid.Text("Press q to quit.", align='center')
+        ])
+        for key, pair in KEY_BINDINGS.items():
+            commands = ""
+            other_key = pair[0]
+            help_text = pair[1]
+            commands += key
+            if key != other_key:
+                commands += ", " + other_key
+            self.log.append(
+                urwid.Columns([
+                    urwid.LineBox(
+                        urwid.Text(commands),
```

Comment:
I like it when the commands are aligned right, so that I can quickly scan what a key does.